osd_quickstart/osd-quickstart.adoc

```diff
@@ -1,20 +1,58 @@
 :_content-type: ASSEMBLY
-[id="osd-quickstart"]
-= Quick start
+[id="osd-getting-started"]
+= Getting started with {product-title}
 include::modules/attributes-openshift-dedicated.adoc[]
-:context: osd-quickstart
+:context: osd-getting-started
 
 toc::[]
 
-// Can the assembly title be more detailed? Plus this uses "Quick start" vs "Quickstart in nav menu"
-
 [role="_abstract"]
-Use this quick start to create and provision a cluster, add users, deploy your first application, and learn how to scale and delete your cluster.
+Follow this getting started document to quickly create a {product-title} cluster, grant user access, deploy your first application, and learn how to scale and delete your cluster.
+
+[id="osd-getting-started-prerequisites_{context}"]
+== Prerequisites
+
+* You reviewed the xref:../osd_architecture/osd-understanding.adoc#osd-understanding[introduction to {product-title}] and the documentation on xref:../osd_architecture/osd-architecture.adoc#osd-architecture[architecture concepts].
+* You reviewed your xref:../osd_cluster_create/osd-understanding-your-cloud-deployment-options.adoc#osd-understanding-your-cloud-deployment-options[{product-title} cloud deployment options].
+
+include::modules/osd-create-cluster-red-hat-account.adoc[leveloffset=+1]
+
```

Comment:
see my comment in the `osd-create-cluster-red-hat-account.adoc` section.  If this is a quickstart why are we only telling users to use a standard red hat account?  I'd expect options should be shown here.

Reply:
@0kashi thank you for this suggestion. We have subsequently reviewed this within the docs team. It might be too much to include procedures for Red Hat cloud account (for which both AWS and GCP can be included in one procedure) as well as the CCS procedures in the getting started page.

The CCS on AWS and CCS on GCP are sufficiently different to warrant two separate procedures. If we include all three procedures in the getting started page, it's not really just a quick page to get the reader into the cluster creation process any more, it's a full section of procedures to cater for all options. That is addressed in the **Creating a cluster** book, which is updated in this PR too.

I realise from a discussion with @arendej earlier this week that CCS is the recommended method, because it enables customers to configure and manage their own cloud accounts in detail with all the features that those bring.

Based on a conversation in our team, we suggest selecting one scenario for the getting started page and then providing a link to the [Understanding your cloud deployment options](https://deploy-preview-41560--osdocs.netlify.app/openshift-dedicated/latest/osd_cluster_create/osd-understanding-your-cloud-deployment-options.html) page for further reading about the other options. That link is already provided in the line below this one in the assembly. I can update the intro to the section that is chosen too, to make this easy to find.

The procedures we could choose one from are as follows:

* Creating a cluster with a Red Hat cloud account
* Creating a cluster on AWS with CCS
* Creating a cluster on GCP with CCS

Reply:
@0kashi, I have now updated this section in the getting started page to provide the option for customers to choose a method from a set of bulleted links. This enables the reader to choose from the available options, as per your suggestion. Please let me know if you have any feedback in relation to this.